heat-templates/openstack-network.yml

```diff
@@ -64,14 +68,14 @@ resources:
       port_security_enabled: False
 
 outputs:
-  network_name:
-    description: Network name
+  network_id:
+    description: Network ID
```

Comment:
I went to look for places we need to update references to these outputs and couldn't find any. Are these three outputs (network, subnet, router) used anywhere?

Reply:
Not yet. But I don't want to remove them because the vars are needed when using terraform to deploy caasp 4